src/javascript/note.js

```diff
@@ -34,7 +34,7 @@ export default class Note {
   }
 
   getDiffPercentage() {
-    if (this.gameHadBeenPaused) {
+    if (this.gameHadBeenPaused || this.vm.playbackSpeed != 1) {
```

Comment:
```suggestion
    if (this.gameHadBeenPaused || this.vm.playbackSpeed !== 1) {
```

See I know JavaScript numbers are IEEE 754 floating-point numbers, subject to implementation details; but is `1.0 === 1`? i.e. could `this.vm.playbackSpeed` take `1` vs `1.0`?

Reply:
Magic